proto/xudrpc.proto

```diff
@@ -377,6 +384,27 @@ message ListPeersResponse {
   repeated Peer peers = 1 [json_name = "peers"];
 }
 
+message ListSwapsRequest {
+  int32 limit = 1 [json_name = "all"];
+  Status status = 3 [json_name = "status"]; 
+  enum Status {
```

Comment:
`Status` is very generic and vague here. Maybe call this `SwapState` which aligns with the enum used within xud, or perhaps even `RequestedSwapState` to be more descriptive.